Yosemite/Yosemite/Stores/PaymentGatewayAccountStore.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+import Networking
+import Storage
+
+/// Implements `PaymentGatewayAccountAction` actions
+///
+public final class PaymentGatewayAccountStore: Store {
```

Comment:
Sorry if you have already considered this. Also, this is a question 😊 

So far, our UI was consuming a WCPayStore, which was pretty much used to capture the payment intent and load the account. 

Are you planning on keeping these two stores (WCPayStore and PaymentGatewayAccountStore) separate? Will those two be merged into one?

Reply:
Long term plan is for PaymentGatewayAccountStore to be the only one - to allow us to support a variety of processors (not jus WCPay). Introducing PaymentGatewayAccountStore is the first step toward eventually decommissioning WCPayStore

Reply:
Sounds good!